test/suites/clustering.sh

```diff
@@ -3172,25 +3172,9 @@ test_clustering_edit_configuration() {
   sed -e "s/:8443/:9393/" -i "${config}"
   LXD_DIR="${LXD_ONE_DIR}" lxd cluster edit < "${config}"
 
-  LXD_DIR="${LXD_TWO_DIR}" lxd cluster show > "${config}"
-  sed -e "s/:8443/:9393/" -i "${config}"
-  LXD_DIR="${LXD_TWO_DIR}" lxd cluster edit < "${config}"
-
-  LXD_DIR="${LXD_THREE_DIR}" lxd cluster show > "${config}"
-  sed -e "s/:8443/:9393/" -i "${config}"
-  LXD_DIR="${LXD_THREE_DIR}" lxd cluster edit < "${config}"
-
-  LXD_DIR="${LXD_FOUR_DIR}" lxd cluster show > "${config}"
-  sed -e "s/:8443/:9393/" -i "${config}"
-  LXD_DIR="${LXD_FOUR_DIR}" lxd cluster edit < "${config}"
-
-  LXD_DIR="${LXD_FIVE_DIR}" lxd cluster show > "${config}"
-  sed -e "s/:8443/:9393/" -i "${config}"
-  LXD_DIR="${LXD_FIVE_DIR}" lxd cluster edit < "${config}"
-
-  LXD_DIR="${LXD_SIX_DIR}" lxd cluster show > "${config}"
-  sed -e "s/:8443/:9393/" -i "${config}"
-  LXD_DIR="${LXD_SIX_DIR}" lxd cluster edit < "${config}"
+  for other_dir in "${LXD_TWO_DIR}" "${LXD_THREE_DIR}" "${LXD_FOUR_DIR}" "${LXD_FIVE_DIR}" "${LXD_SIX_DIR}"; do
+    cp "${LXD_ONE_DIR}/database/recovery_db.tar.gz" "${other_dir}/database/"
+  done
 
```

Comment:
Remove the original `recovery_db.tar.gz` to check that we've done all the needed changes on the generating node without needing it to load the tarball.